scikit_mol/fingerprints.py

```diff
@@ -71,9 +71,24 @@ def fit(self, X, y=None):
         self._get_column_prefix()
         return self
 
+    def _get_mols_from_X(self, X):
+        """Get the molecules iterable from the input X"""
+        if isinstance(X, pd.DataFrame):
+            try:
+                # TODO: possibly handle the case in which a DataFrame
+                # contains multiple molecule columns (as if from a column selector transformer).
+                # In that case, the resulting array should be a concatenation of the fingerprint arrays
+                # for each molecule column.
+                return X.loc[:, "molecule"]
```

Comment:
We can't really be sure that the molecule column is called 'molecule'. It should either be configurable or we should stay with the convention that input to the transformer is always a single column. I think we may be going beyond the scope of the classes by building in column selections to them.